### ISSUE TITLE
Training data set process

### ISSUE THREAD

**Issue by kirankumari999** [goodboyanush/isme-bangalore-Oct-Nov-2019#1]
sir i have doubt in training data process . which you explained on board.
(80*9) sum of w1xi =y^ its predictive value or not.
after that why we are doing (y^(80*1)-y(labels)

**Comment by goodboyanush** [comment]
@kirankumari999 

Step1: we perform *sum of w1xi =y^ * to obtain the predicted output value. However, initially the *w1* could be random weight values, producing random *y^* predicted output values

Step2: We calculate the loss, by computing SquareOf(y^ - y_labels). 

Step3: if the loss value is large, then update *w1* by a large value. If the loss value is small, then update the *w1* by a small value
